daisy_workflows/image_import/ubuntu/translate_ubuntu_2204.wf.json

```
@@ -0,0 +1,72 @@
+{
+  "Name": "translate-ubuntu-2204",
+  "Vars": {
+    "source_disk": {
+      "Required": true,
+      "Description": "The Ubuntu 22.04 GCE disk to translate."
+    },
+    "sysprep": {
+      "Value": "false",
+      "Description": "If enabled, run sysprep. This is a no-op for Linux."
+    },
+    "install_gce_packages": {
+      "Value": "true",
+      "Description": "Whether to install GCE packages."
+    },
+    "image_name": {
+      "Value": "ubuntu-2204-${ID}",
+      "Description": "The name of the translated Ubuntu 22.04 image."
+    },
+    "family": {
+      "Value": "",
+      "Description": "Optional family to set for the translated image"
+    },
+    "description": {
+      "Value": "",
+      "Description": "Optional description to set for the translated image"
+    },
+    "import_network": {
+      "Value": "global/networks/default",
+      "Description": "Network to use for the import instance"
+    },
+    "import_subnet": {
+      "Value": "",
+      "Description": "SubNetwork to use for the import instance"
+    },
+    "compute_service_account": {
+      "Value": "default",
+      "Description": "Service account that will be used by the created worker instance"
+    }
+  },
+  "Steps": {
+    "translate-disk": {
+      "IncludeWorkflow": {
+        "Path": "./translate_ubuntu.wf.json",
+        "Vars": {
+          "ubuntu_release": "focal",
```

Should we update this value as well?
```suggestion
          "ubuntu_release": "jammy",
```